packagehandlers/mavenpackagehandler.go

```diff
@@ -221,7 +221,11 @@ func (mph *MavenPackageHandler) updatePackageVersion(impactedPackage, fixedVersi
 		fmt.Sprintf("-DprocessDependencyManagement=%t", foundInDependencyManagement)}
 	updateVersionCmd := fmt.Sprintf("mvn %s", strings.Join(updateVersionArgs, " "))
 	log.Debug(fmt.Sprintf("Running '%s'", updateVersionCmd))
-	_, err := mph.RunMvnCmd(updateVersionArgs)
+	output, err := mph.RunMvnCmd(updateVersionArgs)
+	if err != nil && strings.Contains(string(output), fmt.Sprintf("Version %s is not available for artifact", fixedVersion)) {
```

Comment:
I suggest making "Version %s is not available for artifact" a global variable and using it in the tests as well. This precaution would ensure that any changes made by Maven to the string are captured in the tests, as we might miss them if the altered string still contains "is not available."

Reply:
when you are right you are not wrong @omerzi